model/model.go

```diff
@@ -287,7 +287,7 @@ func build(kubeClient *client.Client, routerDeployment *extensions.Deployment, p
 			return nil, err
 		}
 		if appConfig != nil {
-			routerConfig.AppConfigs = append(routerConfig.AppConfigs, appConfig)
+			routerConfig.AppConfigs = appConfig
```

Comment:
Still waking this morning and forgot to test locally before pushing this out. Thanks